admin_guide/pruning_resources.adoc

```diff
@@ -451,10 +451,7 @@ times in a single image stream named `myapp`:
 
 ----
 $ image_name="sha:abz"
-$ oc get is -n N -o go-template='{{range $isi, $is := .items}}{{range $ti, $tag := $is.status.tags}}'\
-  '{{range $ii, $item := $tag.items}}{{if eq $item.image "'"${image_name}"\
-  $'"}}{{$is.metadata.name}}:{{$tag.tag}} at position {{$ii}} out of {{len $tag.items}}\n'\
-  '{{end}}{{end}}{{end}}{{end}}'
+$ $ oc get is -n openshift -o go-template='{{range $isi, $is := .items}}{{range $ti, $tag := $is.status.tags}}{{range $ii, $item := $tag.items}}{{if eq $item.image "'$image_name'"}}{{$is.metadata.name}}:{{$tag.tag}} at position {{$ii}} out of {{len $tag.items}}{{end}}{{end}}{{end}}{{end}}'
```

Comment:
> {{end}}{{end}}{{end}}{{end}}' # Before this place {{end}}{{end}}{{end}}{{end}}, use new line

As shown in above comment, need use new line before the place {{end}}{{end}}{{end}}{{end}}, otherwise the output is in one line:
ruby:2.5 at position 0 out of 1ruby:latest at position 0 out of 1

Reply:
> $ $ oc get

Also, two $ typed here

Reply:
Totally missed that. Fixing now. Thanks @xingxingxia!